pkgs/misc/vim-plugins/default.nix

```diff
@@ -941,6 +942,35 @@ rec {
 
   };
 
+  color_coded = buildVimPluginFrom2Nix { # created by nix#NixDerivation
+    name = "color_coded-2017-10-30";
+    src = fetchgit {
+      url = "https://github.com/jeaye/color_coded";
+      rev = "cb356b083362ea6f15bc690c5eb10b01326ad169";
+      sha256 = "0l46ar1b3vy7gbrq2ic734vvw7zcvw6ar78zyhwz55h2lxfh89aq";
+    };
+    dependencies = [];
+    buildInputs = [
```

Comment:
This file is generated, the custom part belongs in `additional-nix-code/color_coded`.

Reply:
I tried that initially and found that my `additional-nix-code/color_coded` wasn't being used at all; didn't find any further documentation and I see other plugins in `default.nix` are specifying their build inputs, so I went ahead with it.

Reply:
It's used when updating all of the plugins, `vim-plugin-names-to-nix` generates the part until `dependencies =[];` and appends the custom snippet afterwards.

Reply:
Yeah, I see the benefit of separating the build bits, since they likely don't change with the auto-generated bits. It's the way that it is because `additional-nix-code/color_coded` didn't work for me and I didn't find any more docs or support elsewhere. If you have any more info, I can try to apply that. If it should *just work* then maybe I need to try it again.